### PR TITLE
New posts

### PR DESCRIPTION
**Pull request by pwalczysko**
See https://trello.com/c/aTdBNblP/18-news-migration under ``Petr`` checklist.

### CODE REVIEW

**Comment by hflynn** [comment]
I've removed all the 'Dear All' and 'Regards' bits so they don't read like emails any more and the old community page links can be updated as internal page links to ({{ site.baseurl }}/support/)

**Comment by pwalczysko** [comment]
@hflynn 
>I've removed all the 'Dear All' and 'Regards' bits 

This means I do not have to do this in this PR because it was done by you somwhere else ?
Or not ?

**Comment by jburel** [comment]
@pwalczysko removed from the ones that have been merged
so you will need to do it on your PR

**Comment by pwalczysko** [comment]
``https://www.openmicroscopy.org/site/support/bio-formats5.1/about/whats-new.html`` is an ``old community page link`` ? And should be replaced by [OME Forums or mailing lists]({{ site.baseurl }}/support/bio-formats5.1/about/whats-new//) ??

File: _posts/2016-04-26-ome-files-cpp-0-1-1.md [hunk 1]
```diff
@@ -83,4 +83,4 @@ Support for building on Windows with *msbuild* is currently non-functional; plea
 
 **Support**
 
-For any problems or comments, please use the OME forums or mailing lists.
+For any problems or comments, please use the [OME Forums or mailing lists]({{ site.baseurl }}/support/)
```
Comment (hflynn) [inline _posts/2016-04-26-ome-files-cpp-0-1-1.md]: This right thanks

File: _posts/2016-05-10-ome-development-status.md [hunk 2]
```diff
@@ -17,4 +17,4 @@ Starting mid-May, the breaking OMERO work initiated on a separate branch will be
 
 [3] https://ci.openmicroscopy.org/view/DEV/
 
-[4] https://www.openmicroscopy.org/site/support/contributing/
+[4] [OME Forums or mailing lists]({{ site.baseurl }}/support/contributing/)
```
Comment (hflynn) [inline _posts/2016-05-10-ome-development-status.md]: This is a link to the contributing docs and should be left as is for now

File: _posts/2016-05-10-omero-5-2-3.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ System administrator updates include:
 -  warnings added regarding the end of Windows support in the 5.3.0 release`[1]` (note that we will be preparing a guide for migrating from Windows for existing servers and adding it to the documentation as soon as we can)
 
 This release also upgrades the version of Bio-Formats which OMERO uses to
-5.1.9. For more information see https://www.openmicroscopy.org/site/support/bio-formats5.1/about/whats-new.html.
+5.1.9. For more information see [What's new]({{ site.baseurl }}/support/bio-formats5.1/about/whats-new/).
```
Comment (hflynn) [inline _posts/2016-05-10-omero-5-2-3.md]: this is a docs link and can't use the 'site.baseurl' - just keep the full old URL for now

**Comment by hflynn** [comment]
I'm going to have to grep and sort out the old website links anyway so I'll just merge this as is, thanks @pwalczysko

**Pull request merged**